cmd/storage-datatypes.go

```diff
@@ -32,6 +32,7 @@ type DiskInfo struct {
 	Free       uint64
 	Used       uint64
 	UsedInodes uint64
+	FreeInodes uint64
```

Comment:
The API version must be bumped as per the comment.

Reply:
done